### DISCUSSION TITLE
ngx-mat-tui-calendar (Angular component for TUI Calendar) released

### DISCUSSION THREAD

**Discussion by ron2015schmitt** [nhn/tui.calendar#959] [Show and tell]
I created an Angular Material Design wrapper for TUI Calendar with buttons and theming.

[online demo](https://ngx-mat-tui-calendar-demo.netlify.app/)

[npm](https://www.npmjs.com/package/ngx-mat-tui-calendar)
[github](https://github.com/ron2015schmitt/ngx-mat-tui-calendar)

**Comment by adhrinae** [comment]
It looks nice! But you might need to check the layout problem when I move to prev/next in the week view or the day view.

**Reply by ron2015schmitt** [reply]
I'm glad you like it. I'll fix the prev/next issue and get back to you here.  Good find.  This worked at one point.

Can you put this as a/the link for Angular in the TUI Calendar documentation/README?

**Reply by adhrinae** [reply]
Yes, I will. Or You can upload a PR about that.

**Reply by ron2015schmitt** [reply]
Ok, I have the arrow button issues fixed.  I will put in a PR for the README change....
